InteractionBase/CMakeLists.txt

```diff
@@ -192,4 +192,4 @@ add_library(InteractionBase SHARED
 	src/handlers/HMovableItem.cpp
 )
 
-envision_plugin(InteractionBase VisualizationBase FilePersistence)
\ No newline at end of file
+envision_plugin(InteractionBase VisualizationBase FilePersistence VersionControlUI)
```

Comment:
This is doubly broken since version control UI also depends on interaction base. This is a circular dependency !!!

You would have noticed that if you correctly added the dependency here, by also including it in the `.plugin` file. Then neither of those plugins would have loaded and you would have gotten an error message. The code works right now on a system like linux, which has relaxed requirements for dependencies, but you wouldn't be able to compile it on windows for example.